src/Handler.php

```diff
@@ -69,15 +137,23 @@ public function exceptionHandler($throwable)
      */
     public function errorHandler($errno, $errstr, $errfile = '', $errline = 0)
     {
-        if ($this->client->shouldIgnoreErrorCode($errno)) {
-            return;
-        }
```

Comment:
> FWIW, the interpreter interprets this null return value differently than the false you were returning below.

Looks like a bug. Either our doc was wrong, and we meant `bool|null`, or we meant to `return false`.